docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,84 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** saves an image to either **docker-archive**, **oci-archive**, **oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type) on the local machine, and then copies the saved image to a remote host specified either by **--remote** globally, or by a connection name which has been added using **podman system connection add**. if neither **--remote** or a **HOSTNAME** has been specified, **image scp** will execute a local copy and save. default format is **docker-archive**. **podman image scp** writes to STDOUT by default and can be redirected to a file using the **output** flag. The **quiet** flag suppresses the output when set. **podman image scp** utilizes **podman save** and **podman load on the specified host or locally**.
```

Comment:
This opening paragraph is very confusing to me,  reading it I have very little idea what the command is supposed to do, I don't even understand the options.

Let's agree on the man page, then make the code match it.  The man page for this command is it's design document

```
**podman image scp** copies container images between hosts on a network.
```